app/helpers/application_helper.rb

```diff
@@ -492,10 +492,8 @@ def accessible_resource(obj, resource, order = :name, association: resource)
     list
   end
 
-  def accessible_related_resource(obj, relation, opts = {})
+  def accessible_related_resource(obj, relation, order: :name, where: nil)
     return [] if obj.blank?
-    order = opts.fetch(:order, :name)
-    where = opts.fetch(:where, nil)
```

Comment:
not sure I understand your comment here, the behaviour doesn't change at all, just replace the ops hash into kwargs.